translations/ModulesAutoupgradeAdmin.en.xlf

```diff
@@ -26,11 +26,6 @@
   </file>
   <file original="classes/Parameters/LocalChannelConfigurationValidator.php" source-language="en-US" target-language="en-US" datatype="plaintext">
     <body>
-      <trans-unit id="8cd1c74d3c55f934c1b312fc65f9022f">
-        <source>We couldn't find a PrestaShop version in the .zip file that was uploaded in your local archive. Please try again.</source>
-        <target>We couldn't find a PrestaShop version in the .zip file that was uploaded in your local archive. Please try again.</target>
-        <note>Line: 144</note>
-      </trans-unit>
```

Comment:
That's weird, this text should have remained in the file because it still exist in the code base

https://github.com/PrestaShop/autoupgrade/blob/08e1296bdd345d3be9d09f5b715aaa4be4c856b9/classes/Parameters/LocalChannelConfigurationValidator.php#L144

Looks like the translation keeps only one occurence in the translation file.

https://github.com/PrestaShop/autoupgrade/blob/08e1296bdd345d3be9d09f5b715aaa4be4c856b9/classes/Task/Miscellaneous/UpdateConfig.php#L102

We need to check the translation can be applied everywhere

Reply:
Whatever the place the translation is, it will spread accross files. There is one improvement to avoid the changes on each PR